lib/maze/schemas/trace_validator.rb

```diff
@@ -25,8 +25,6 @@ def initialize(body)
       # Runs the validation against the trace given
       def validate
         @success = true
-        # Shortcut the validation if the body is empty for initial P gathering reasons
-        return if @body.keys.eql?(['resourceSpans']) && @body['resourceSpans'].empty?
```

Comment:
This is no longer needed thanks to the code in `validator.rb`.  `validator_class` will be `nil` for p-value requests as they are now held on a separately named list.